python/paddle/fluid/imperative/nn.py

```diff
@@ -432,13 +434,14 @@ def __init__(self,
                  is_distributed=False,
                  padding_idx=None,
                  param_attr=None,
+                 stop_gradient=False,
                  dtype='float32'):
 
         super(Embedding, self).__init__(name_scope)
         self._size = size
         self._is_sparse = is_sparse
         self._is_distributed = is_distributed
-
+        self._stop_gradient = stop_gradient
```

Comment:
one_hot should not have gradient and if we don't set this it will cause segment fault in imperative mode, better option is set all op to stop gradient first and then change to false case by case